packages/kit-bg/src/services/ServiceAppCleanup/ServiceAppCleanup.ts

```
@@ -100,12 +104,15 @@ class ServiceAppCleanup extends ServiceBase {
   }
 
-  async cleanupIndexedAccounts() {
+  async cleanupIndexedAccounts(indexedAccountsRemoved?: IDBIndexedAccount[]) {
     await this.runCleanupTask(async () => {
-      const { indexedAccountsRemoved } =
-        await this.backgroundApi.serviceAccount.getAllIndexedAccounts({
-          filterRemoved: true,
-        });
+      if (!indexedAccountsRemoved) {
+        // eslint-disable-next-line no-param-reassign
+        ({ indexedAccountsRemoved } =
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)